scalyr_agent/copying_manager/copying_manager.py

```diff
@@ -146,6 +156,27 @@ def api_key_id(self):
     def workers(self):
         return self.__workers
 
+    def get_child_process_ids(self):
+        # type: () -> List[str]
+        """
+        Return a list of all the children processes process ids (pids) when running in multiprocess
+        mode.
+
+        When multi process mode is not used, empty list is returned instead.
+        """
+        if not self.__shared_object_managers:
+            return []
+
+        result = []
+
+        for shared_object_manager in self.__shared_object_managers.values():
+            try:
+                result.append(shared_object_manager._process.ident)
+            except Exception:
+                pass
```

Comment:
Technically it's an internal non-public API attribute so I added it as a safe guard for just in case (and IIRC, it's not available in Python 2.6).

I can add the comment to that line.

Reply:
The only reason that I see is if `shared_object_manager` does not have a `_process` attribute.  I guess Tomaz had meant this.

Reply:
Yep and as a safe-guard for "future proofing" since we are accessing "internal" attribute (sadly there is no public interface to get this info, afaik).

Reply:
@Kami sorry, I haven't seen your first comment. It hasn't been shown without page refresh for some reason.